src/api/spec/controllers/webui/cloud/azure/upload_jobs_controller_spec.rb

```diff
@@ -0,0 +1,230 @@
+require 'rails_helper'
+require 'webmock/rspec'
+
+RSpec.describe Webui::Cloud::Azure::UploadJobsController, type: :controller, vcr: true do
+  let(:azure_configuration) { build(:azure_configuration, application_id: 'Hey OBS!', application_key: 'Hey OBS?') }
+  let(:user_with_azure_configuration) { create(:confirmed_user, login: 'tom', azure_configuration: azure_configuration) }
+  let(:project) { create(:project, name: 'Apache') }
+  let!(:package) { create(:package, name: 'apache2', project: project) }
+  let(:upload_job) { create(:upload_job, user: user_with_azure_configuration) }
+  let(:xml_response) do
```

Comment:
Can equally be a file fixture...

Reply:
We already do this on several other places. Maybe we should talk about how we want to do that...